package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@percy/selenium-webdriver",
   "description": "Selenium client library for visual testing with Percy",
-  "version": "2.0.4",
+  "version": "2.0.4-beta.0",
```

Comment:
I had already changed that to 2.0.4 version, but as per the process it should first to 2.0.4-beta.0 and then to 2.0.4. 
This can be ignored